Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG CLI_VERSION=latest
```

Comment:
Rather than specifying the argument in here, can we define it as a build parameter and pull in here, or is this exactly how it works and the value here is merely the default in case no value has been specified?

Reply:
This acts as a default value, if an arg is not passed, so using `--build-arg CLI_VERSION={version}` in the Docker build command replaces this value allowing us to specify package versions at build time.